unit-testing-react-what-why-how/Button/solution/package.json

```diff
@@ -9,20 +9,24 @@
   "dependencies": {
```

Comment:
Should we add jest/types to these dependencies?
everything works as expected, but I"m getting errors that `describe` isn't defined, even though I can run the tests just fine.
<img width="896" alt="image" src="https://github.com/bitovi/trainings/assets/65362632/816c47bb-a9cb-4e1a-9622-430e56e5cb8d">

Reply:
Same as this: https://github.com/bitovi/trainings/pull/37#discussion_r1348176714